server/sonar-web/src/main/less/components/columns.less

```diff
@@ -32,6 +32,13 @@
   overflow: visible !important;
 }
 
+.column-full {
```

Comment:
I think it's better to not add utility css classes to the sonarqube. It'll be hard to know if they are still used in future.

So please move them to the plugin, and moreover prefix them with `billing-` or something similar to avoid collisions.